src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.1.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
-    <NuGetTargetMoniker>.NETPlatform,Version=v5.4</NuGetTargetMoniker>
+    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
```

Comment:
We reviewed this change previously with @mellinoe and determined this was not necessary.  Has the thinking here changed?